src/OmniSharp.Abstractions/Options/CodeActionOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace OmniSharp.Options
+{
+    public class CodeActionOptions
+    {
+        public string[] LocationPaths { get; set; }
+
+        public IEnumerable<string> GetLocations(IOmniSharpEnvironment env)
```

Comment:
Perhaps this should be called something like `GetNormalizedLocationPaths(...)` or something like that to indicate what it does differently over just accessing the `LocationPaths` property?